client/src/components/DataFiles/DataFiles.js

```diff
@@ -114,11 +114,11 @@ const DataFiles = () => {
   const loading = useSelector(state => state.systems.storage.loading);
   const error = useSelector(state => state.systems.storage.error);
   const systems = useSelector(state => state.systems.storage.configuration);
-
+  var label = "DATA"
```

Comment:
@fnets changes look good, should we fix the linting errors before this gets merged?